error/error_handler.go

```diff
@@ -119,17 +158,22 @@ func EchoHTTPErrorHanderFunc(e error, c echo.Context) (bool, error) {
 			}
 		}
 
-		// Get from env.json file.
-		html504File := viper.GetString("http.timeoutMessage.html.file")
-
 		if !strings.Contains(c.Request().Header.Get("Content-Type"), "application/json") {
+			// Get from env.json file.
+			html504File := viper.GetString("http.errorMessage.e504.html.file")
 			if html504File != "" {
 				err = c.Render(he.Code, html504File, echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
 			} else {
 				err = c.Render(he.Code, "errors/html/504", echo.Map{"stacktrace": fmt.Sprintf("%+v", e)})
 			}
 		} else {
-			err = c.JSON(he.Code, he.Message)
+			// Get from env.json file.
+			e504 := viper.GetStringMap("http.errorMessage.e504")
+			if val, ok := e504["json"]; ok {
+				err = c.JSON(he.Code, val)
+			} else {
+				err = c.JSON(he.Code, errorResp{ErrorCode: TIMEOUT, ErrorMsg: "gateway timeout"})
```

Comment:
the processing logic of `ErrorMsg: "gateway timeout"` is different from other places，confusing..